circle.yml

```diff
@@ -1,11 +1,15 @@
 version: 2
 
 .x-cockroach-2.1: &cockroach-2-1
-  image: cockroachdb/cockroach:v2.1.9
+  image: cockroachdb/cockroach:v2.1.11
   command: start --insecure
 
 .x-cockroach-19.1: &cockroach-19-1
-  image: cockroachdb/cockroach:v19.1.5
+  image: cockroachdb/cockroach:v19.1.8
+  command: start --insecure
+
+.x-cockroach-19.1: &cockroach-19-2
```

Comment:
Looks like you have -19.1 defined twice.

Reply:
whoops. thanks for the catch!